aioesphomeapi/connection.py

```diff
@@ -170,7 +170,7 @@ def __init__(
         self._connect_complete = False
 
         # Message handlers currently subscribed to incoming messages
-        self._message_handlers: Dict[Any, List[Callable[[message.Message], None]]] = {}
+        self._message_handlers: Dict[Any, Set[Callable[[message.Message], None]]] = {}
```

Comment:
If the set slows down the callback, revert that and remove the suppress and switch to a bare try

Reply:
its fine